docs/release-notes/8.4.0-4.md

```diff
@@ -0,0 +1,26 @@
+# Percona XtraBackup 8.4.0-4 (2025-08-12)
+
+Get started with [Quickstart Guide for Percona XtraBackup].
+
+## Release highlights
+
+This release provides improvements and bug fixes.
+
+We recommend that you download the Percona XtraBackup for the same platform as the MySQL-compatible server. For example, if your server is on an ARM64 platform, you should download and use the Percona XtraBackup for ARM64 for that operating system.
+
+## Bug fixes
+
+[PXB-3571](https://perconadev.atlassian.net/browse/PXB-357): When you use transition keys with `--lock-ddl=reduced`, XtraBackup does not save the encrypted tablespace keys in the `xtrabackup_keys` file.
+
+[PXB-3470](https://perconadev.atlassian.net/browse/PXB-3470): Xbstream spends a significant amount of time on post-processing after receiving streamed backups that contain huge tables.
```

Comment:
If a backup is taken with --compress (lz4 or ztd) and is streamed (either to object storage or to local .stream file), xbstream --decompress or xtrabackup --decompress took significant time to decompress the backups